css/style.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+body {
+  position: relative;
+}
+
+section {
+  width: 100vw;
+  margin: 0;
+}
+
+#navbar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 80px;
+  width: 100vw;
+  background-color: #be3144;
+  display: flex;
+  z-index: 10;
+}
+
+.nav-list {
+  margin-left: auto;
+  display: flex;
+  align-items: center;
+}
+
+.nav-list > li {
+  display: inline-block;
+  margin-right: 25px;
+}
+
+.nav-list a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 1rem;
+}
+
+#welcome-section {
+  top: 80px;
+  height: 100vh;
+  background-color: #2f3034;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#welcome-section > h1 {
+  color: #fff;
+  font-size: 2rem;
+  margin-bottom: 5px;
+}
+
+footer > p {
+  color: #fff;
+}
+
+#welcome-section > p {
+  color: #be3144;
+  font-size: 1.5rem;
+}
+
+#projects {
+  background-color: #45567d;
+  padding: 80px 30px;
+}
+
+#projects > h1 {
+  text-align: center;
+  margin-bottom: 50px;
+  color: #fff;
+  font-size: 1.8rem;
+  text-decoration: underline;
+}
+
+.projects-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 4rem;
+  width: 100%;
+  padding: 0;
+}
+
+.projects-grid a {
+  text-decoration: none;
+}
+
+.project-tile {
+  background-color: #303841;
+  list-style: none;
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
+  border-radius: 2px;
+}
+
+.project-title {
+  padding: 1.5rem 0.5rem;
+  color: #fff;
+  text-align: center;
+  font-size: 2rem;
+}
+
+.project-img {
+  max-width: 100%;
+  width: 100%;
+}
+
+#contact {
+  text-align: center;
+  height: calc(100vh - 80px);
+  padding: 25px;
+  background-color: #303841;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ] [REQUIRED] It will be nice if you don't repeat the same CSS rules. DRY and Immutable code is your endpoint in developing your Infrastructure-as-Code deployments. Working towards it will improve your environment, making your code more readable and manageable.

Reply:
You have repeated this same comment up to 8 times. Again, just like I mentioned before, if the mistake is being repeated multiple times, you don't have to select all occurrences, just select it once, and tell them to fix every other one.